apps/prairielearn/src/pages/instructorGradingJob/instructorGradingJob.html.ts

```diff
@@ -0,0 +1,210 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { z } from 'zod';
+
+const GradingJobSchema = z.object({
+  id: z.number(),
```

Comment:
This should use `IdSchema` instead of `z.number()`.

Reply:
Resolved in commit 4ce99d4.